.gitignore

```diff
@@ -25,7 +25,11 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
-
+test.py
+.idx
+.vscode
+.venv
+.github
```

Comment:
Can we add another section for these in the `.gitignore`. Im thinking something like?

```suggestion
# Misc config stuff

.idx
.vscode
.github
```

2 other things though:
* `.venv` should already be covered in the `# Enviornments` section so we can remove that.
* Can we remove `test.py` since that isn't a part of the repo itself atm?

Reply:
Changes made.  Please let me know if you have any other requests.

Reply:
Looks great! Thanks for contributing!